Tests/scripts/stop_running_pipeline.py

```diff
@@ -112,6 +119,7 @@ def cancel_pipelines_for_branch_name(gitlab_client: Gitlab,
                 logging.info(f"Trying to cancel pipeline for test upload flow branch:{test_upload}")
                 success &= cancel_pipelines_for_branch_name(gitlab_client, project, test_upload, "trigger",
                                                             instances_service, all_instances, False)
+                delete_test_upload_flow_branch(project, test_upload)
```

Comment:
only cancel in case it's a "child" pipeline, it means it's a test_upload_flow branch and build.

Reply:
Not sure I understand this case. Isn't the after script should handle all cases?